onnxruntime/core/providers/cpu/ml/tree_ensemble_classifier.cc

```diff
@@ -174,6 +174,7 @@ common::Status TreeEnsembleClassifier<T>::Compute(OpKernelContext* context) cons
   Tensor* Y = context->Output(0, TensorShape({N}));
   Tensor* Z = context->Output(1, TensorShape({N, tree_ensemble_.get_class_count()}));
 
+  ORT_ENFORCE(nullptr != Y && nullptr != Z, "outputs must not be NULL");
```

Comment:
>ORT_ENFORCE(nullptr != Y && nullptr != Z, "outputs must not be NULL"); [](start = 2, length = 70)

These outputs are not options, are they? This check is not required.

Reply:
Agree, removing the line.